substrate/client/rpc-spec-v2/src/archive/api.rs

```diff
@@ -36,8 +36,8 @@ pub trait ArchiveApi<Hash> {
 	/// # Unstable
 	///
 	/// This method is unstable and subject to change in the future.
-	#[method(name = "archive_unstable_body")]
-	fn archive_unstable_body(&self, hash: Hash) -> RpcResult<Option<Vec<String>>>;
+	#[method(name = "archive_v1_body")]
+	fn archive_v1_body(&self, hash: Hash) -> RpcResult<Option<Vec<String>>>;
```

Comment:
We shouldn't return RpcResult from this API IMHO, which means that one can return any JSON-RPC error object.

Really, it should be either a concrete error enum which possible states or just `Option<Vec<String>>` if infallible the same applies to all other rpc impls below.

Ah, we are missing an impl for Option<T> for IntoResponse and probably all these infallible.

However, before stabilization it would be good clean that up

Reply:
ah damn, it doesn't work when the client stuff is enabled in rpc macro never mind then